NitroxClient/GameLogic/Entities.cs

```diff
@@ -106,13 +102,18 @@ private void UpdatePosition(Entity entity)
 
             if (opGameObject.IsPresent())
             {
-                opGameObject.Get().transform.position = entity.Position;
-                opGameObject.Get().transform.localRotation = entity.Rotation;
-                opGameObject.Get().transform.localScale = entity.Scale;
+                opGameObject.Get().transform.localRotation = entity.LocalRotation;
+                opGameObject.Get().transform.localPosition = entity.LocalPosition;
+                opGameObject.Get().transform.localScale = entity.LocalScale;
+                if (entity.LocalPosition != new Vector3(0, 0, 0))
+                {
+                    opGameObject.Get().transform.position = entity.Position;
```

Comment:
I'm not entirely understanding this area.  We only update the position if there was a local position?  Wouldn't this cause a lot of entities to not be updated?

Reply:
Was supposed to be entity.Position...